src/containers/BookForm.js

```diff
@@ -1,18 +1,54 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import {createBook} from '../actions/index';
 
 
 const Categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'] 
 class BookForm extends React.Component {
+	state = {
+		title : '',
+		category : Categories[0],
+	}
+	handleChange = (e) => {
+      this.setState({
+         [e.target.name]: e.target.value
+      });
+	}
+	handleSubmit = (e) => {
+		e.preventDefault();
+		const { title, category } = this.state;
+		this.props.addItem({ title, category });
+		this.setState({
+                title: '',
+        })  
+	}
```

Comment:
Hi, After submit, let's clear the form. `e.target.reset()` should do.
you also didn't bind the value of title to the title in the form so even though you setState of title to ' ', it won't reflect.